validate.sh

```diff
@@ -18,6 +18,11 @@ dydxprotocold_path=$(tar -xvf $tar_path --directory /tmp/dydxprotocold)
 mkdir -p ~/bin
 cp /tmp/dydxprotocold/$dydxprotocold_path /usr/local/bin/dydxprotocold
 
+NINE_ZEROS="000000000"
+EIGHTEEN_ZEROS="$NINE_ZEROS"
+INITIAL_VALIDATOR_TOKEN_BALANCE="1000000$EIGHTEEN_ZEROS"
+INITIAL_VALIDATOR_SELF_DELEGATION="500000$EIGHTEEN_ZEROS"
```

Comment:
This requires `v0.4.0-rc0` binary to be accepted by the validation workflow @2dpetkov